WcaOnRails/app/models/competition.rb

```diff
@@ -824,6 +824,25 @@ def to_wcif
     }
   end
 
+  def load_wcif!(wcif)
```

Comment:
I like your new proposed name for the method, although I would argue that this method *does* in fact "store a WCIF JSON document to the DB", it's just an implementation detail that it gets stored in tables.